gammapy/modeling/models/temporal.py

```diff
@@ -657,6 +683,53 @@ def to_dict(self, full_output=False, format="table"):
         data["temporal"]["unit"] = str(self.map.unit)
         return data
 
+    def plot(self, time_range, energy=None, ax=None, n_points=100, **kwargs):
+        """
+        Plot Temporal Model.
+
+        Parameters
+        ----------
+        time_range : `~astropy.time.Time`
+            times to plot the model
+        energy : `~astropy.units.quantity`
```

Comment:
I would move the `energy` argument to later to follow the arguments of `TemporalModel.plot`